src/sentry/relocation/tasks/process.py

```diff
@@ -1442,6 +1519,12 @@ def importing(uuid: UUID) -> None:
     retry_backoff_jitter=True,
     soft_time_limit=FAST_TIME_LIMIT,
     silo_mode=SiloMode.REGION,
+    taskworker_config=TaskworkerConfig(
+        namespace=relocation_tasks,
```

Comment:
Should we have a processing_deadline of FAST_TIME_LIMIT to align with the celery configuration?